indico/modules/search/client/js/components/results/Contribution.module.scss

```diff
@@ -0,0 +1,61 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2019 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+@import 'base/palette';
+
+$darken-parameter: 50;
+
+.contribution {
+    padding: 1em 0;
+    color: $dark-gray;
+
+    .description {
+        padding: 0.4em 0;
+
+        a , a:hover {
+            font-size: 15px;
+            color: $dark-gray;
+        }
+
+        .high-priority {
```

Comment:
Maybe this part starts getting a little bit too nested? It's not that good for readability.

Reply:
I'm trying to figure out how not to use nesting in css